### PR TITLE
[WIP] Update homepage design

### PR DESCRIPTION
**Pull request by line47**
### Screenshot 
localhost-1337-

Updates:
- Using a single `log in` button and changing the link in the top nav bar to be a button.
- Update the hero unit and text to encourage signup for more info instead of log in.
- 3 column layout to show the high level features 
- Simplify and call attention to the open for business section (also wondering if that content would be moved up into the hero)

### CODE REVIEW

**Comment by line47** [comment]
@jseppi I need some help finding the tests to update in order to get this passing circleci

**Comment by jseppi** [comment]
Sure, when you run `yarn test` locally, you'll see output like this:

```sh
 243 passing (10s)
  1 failing

  1) Main Site Home / should display the number of builds from the past week:
     AssertionError: expected -1 to be above -1
      at Build.destroy.then.then.then (test/api/requests/main.test.js:68:105)
      at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
      at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
      at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
      at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
      at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
      at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
      at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
      at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/release/async.js:17:14)
```

The first place it gives a file path is where you'll likely find the failing test: `test/api/requests/main.test.js:68:105` -- that means line 68, column 105 in `test/api/requests/main.test.js`.

When we open that file, we see this on line 68:

```js
expect(response.text.indexOf('Federalist has deployed 7 site updates in the past week.')).to.be.above(-1);
```

That test is looking for specific text, but in the design you've changed the wording a bit. You'll just have to update the text it is looking for to be consistent with the new text. ie,

```js
expect(response.text.indexOf('7 site updates deployed in the past week.')).to.be.above(-1);
```

**Comment by jseppi** [comment]
You'll also see the same test information in CircleCI if you click on the "Details" link where it says "ci/circleci -- Your tests failed on CircleCI"

screen shot 2017-11-17 at 9 40 20 am

...scroll waaaay down

screen shot 2017-11-17 at 9 40 43 am

File: test/api/requests/main.test.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ describe('Main Site', () => {
           .expect(200)
       )
       .then((response) => {
-        expect(response.text.indexOf('Federalist has deployed 7 site updates in the past week.')).to.be.above(-1);
+        expect(response.text.indexOf('7 site updates in the past week.')).to.be.above(-1);
```
Comment (jseppi) [inline test/api/requests/main.test.js]: I think you missed the word "deployed":

`7 site updates deployed in the past week.`

Comment (line47) [reply test/api/requests/main.test.js]: Thanks!

File: frontend/sass/_text.scss [hunk 2]
```diff
@@ -2,7 +2,32 @@ body {
   color: $text-color;
   font-family: $sans-serif;
 }
+// Scott start styles
```
Comment (el-mapache) [inline frontend/sass/_text.scss]: do we need to keep these wrapper comments around these styles?

**Comment by el-mapache** [comment]
This is definitely an improvement over the previous home page, thanks for reworking the design (and cleaning up all the inline svgs 😬 )!

File: frontend/sass/_text.scss (hunk 2 above)
Comment (line47) [reply frontend/sass/_text.scss]: No - that was for my WIP, thanks for finding that. I'd like to move the styles where they belong.

**Comment by line47** [comment]
love to get this merged this week @wslack - we can revisit once we get more visual design direction. 👍

**Comment by jseppi** [comment]
Is this still a WIP, @line47?

**Comment by line47** [comment]
We were waiting on some designs (which just came over from @ericronne this morning!) so maybe we should incorporate those ?

**Comment by wslack** [comment]
Let's go ahead and merge as is into staging then work on another PR with what Eric just turned in.

File: frontend/sass/_text.scss (hunk 2 above)
Comment (jseppi) [reply frontend/sass/_text.scss]: @line47 - can you make a quick PR to remove these comments and move the styles to where they belong?

**Pull request merged**